src/chess/pgn.h

```diff
@@ -70,10 +70,14 @@ class PgnReader {
       throw Exception(errno == ENOENT ? "Opening book file not found."
                                       : "Error opening opening book file.");
     }
+
     std::string line;
     bool in_comment = false;
     bool started = false;
     while (GzGetLine(file, line)) {
+      // Check if we have a UTF-8 BOM. If so, just ignore it.
+      // Only supposed to exist in the first line, but should not matter.
+      if (line.find("\xEF\xBB\xBF", 0) == 0) line = line.substr(3);
```

Comment:
Actually `if (line.substr(0,3) == "\xEF\xBB\xBF")` sounds like a better idea (until we have C++20 line.starts_with()), as otherwise it scans entire line while it only needs the beginning of it.